onnx/test/reference_evaluator_backend_test.py

```diff
@@ -461,7 +481,7 @@ class TestOnnxBackEndWithReferenceEvaluator(unittest.TestCase):
 
     @classmethod
     def add_test_methods(cls):
-        for folder in ["node", "pytorch-converted", "pytorch-operator", "simple"]:
+        for folder in ["node"]:
```

Comment:
Why this change?

Reply:
My mistake. I commented for debugging. Forgot to bring it back.